include/swift/AST/DiagnosticsSema.def

```diff
@@ -3516,6 +3516,10 @@ WARNING(pbd_never_used_stmtcond, none,
         "value %0 was defined but never used; consider replacing "
         "with boolean test",
         (Identifier))
+WARNING(unused_implicit_setter_newvalue, none,
+        "implicit setter argument 'newValue' is never used; consider specifying"
+        " a variable name or using the argument `newValue`",
```

Comment:
Great, much better, thanks @DougGregor! Thanks for the feedback too -- glad to get a re-direct before changing the 100 tests ;0